relay-general/src/pii/event_paths.rs

```diff
@@ -0,0 +1,210 @@
+use std::collections::BTreeSet;
+
+use crate::processor::{
+    process_value, Pii, ProcessValue, ProcessingState, Processor, SelectorPathItem, SelectorSpec,
+    ValueType,
+};
+use crate::protocol::{AsPair, Event, PairList};
+use crate::types::{Annotated, Meta, ProcessingResult};
+
+use crate::pii::processor::process_pairlist;
+
+struct EventPathsProcessor {
```

Comment:
A note for later: This processor is independent of events, so it could be named independently. Then we wouldn't have a dependency on the `Event` struct, which would further simplify the dependency graph and provide for decoupling the `PII` module from `protocol` and `types`, which will be the ultimate goal.

Until we change processor derives, we're still quite far from this, however.